OxfordBodleian/Juel-Jensen/BDLaethe32.xml

```diff
@@ -189,21 +636,221 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                           
+                                <date notBefore="1649" notAfter="1849"/>
+                                <desc>Irregular, but probably trained, hand. Letters are spaced irregularly, and written lines placed also somewhat irregularly
+                                on the ruled lines.</desc>
+                                <seg type="ink">Black, red</seg>
+                                <seg type="rubrication">
+                                    Several groups of lines on the incipit pages, alternating with black lines; titles of the psalms and of the
+                                    canticles; incipits of the songs; the name <foreign xml:lang="gez">ማርያም፡</foreign> and its abbreviations;
+                                    name and traditional interpretation of the Hebrew letters in Ps 118; the abbreviation standing for the refrain 
+                                    in <ref target="#ms_i1.4"/> and <ref target="#ms_i1.5"/>;
+                                 elements of the punctuation signs, text dividers, and numerals.
+                                </seg>
+                                <list type="abbreviations">
+                                    <item>Single letters for the entire refrain in <ref target="#ms_i1.4"/> and <ref target="#ms_i1.5"/>;</item>
+                                    <item>
+                                        <abbr>ሰ፡ ለ፡ ቅ፡</abbr>  for <expan>ሰአሊ፡ ለነ፡ ቅድስት፡</expan> (e.g. <locus target="#109rb #114ra #117ra"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ማርያ፡</abbr> and   <abbr>ማር፡</abbr>  for <expan>ማርያም፡</expan> (e.g. <locus target="#112rb #114ra #114va"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ወ፡ ሣ፡ ይ፡ ሰ፡ ለ፡ ቅ፡</abbr> for <expan>ወልድኪ፡ ሣህሎ፡ ይክፍለነ፡ ሰአሊ፡ ለነ፡ ቅድስት፡</expan> (e.g. <locus target="#120ra #120vb"/>)
+                                    </item>
+                                </list>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
                          
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1r"/>
+                                    <desc type="Unclear">Pen trial followed by a list in Amharic, hardly legible.</desc>
+                                    <q xml:lang="am"/>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#1v"/>
+                                    <desc type="MagicText">ʾAsmāt-prayer</desc>
```

Comment:
It would be more logical, yes. I think what will count for us will be that they will be printed in italics, right? Since we have no way yet to mark up the difference between Geez in fidal and Geez in transliteration with `<foreign>`, adding it won't help in this regard, but at least we'll be more consistent. (We'll probably add the italics for these cases manually later)